references/detection/train_tpu.py

```diff
@@ -98,7 +119,7 @@ def random_colour_masks(mask):
 
 
 def overlay_output(img, masks, boxes, pred_cls, alpha=0.5):
-    for i in range(len(masks)):
+    for i in range(len(boxes)):
```

Comment:
is `boxes` of different length sometimes? is this intended?

Reply:
No, I had made that change when debugging the masks as I thought it might be that `len(boxes) != len(masks)`, but it's never the case (rightfully so) and I forgot to revert this line back to the original. I can revert it in a new commit if you want, but the output will be the same 😃

Reply:
yeah you could push a new commit to this branch and it'll update the pr.